stl/inc/xutility

```diff
@@ -21,11 +21,13 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma push_macro("new")
 #undef new
 
+#ifndef _USE_STD_VECTOR_ALGORITHMS
 #if (defined(_M_IX86) || defined(_M_X64)) && !defined(_M_CEE_PURE) && !defined(_M_HYBRID) && !defined(_M_ARM64EC)
 #define _USE_STD_VECTOR_ALGORITHMS 1
 #else
 #define _USE_STD_VECTOR_ALGORITHMS 0

```

Comment:
Do you think we should restructure this so we can emit a message if `_USE_STD_VECTOR_ALGORITHMS` is set to `1` on an architecture for which the vector versions aren't implemented? Escape hatches are necessarily tools with sharp edges, but with this implementation such an erroneous usage will manifest at link time as missing symbols with "weird" names that users might not recognize as being related to `_USE_STD_VECTOR_ALGORITHMS`.

(This does look fine to me otherwise.)